src/config.rs

```diff
@@ -51,3 +52,19 @@ pub fn zo_maxage() -> Result<Rank> {
         None => Ok(1000.0),
     }
 }
+
+pub fn zo_fzf_extra_args() -> Result<Vec<String>> {
+    match env::var_os("_ZO_FZF_EXTRA_ARGS") {
```

Comment:
Something like this would save you from the layers of nesting:

```rust
match env::var("_ZO_FZF_EXTRA_ARGS") {
    Ok(fzf_args_str) => {
        shlex::split(fzf_args_str).ok_or_else(anyhow!("could not parse _ZO_FZF_EXTRA_ARGS"))
    }
    Ok(env::VarError::NotPresent) => Ok(Vec::new()),
    e @ Err(env::VarError::NotUnicode) => e.context("invalid Unicode in _ZO_FZF_EXTRA_ARGS"),
}
```